src/App.tsx

```diff
@@ -15,48 +17,119 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortField {
+  DEFAULT = '',
+  ALPHABETICALLY = 'alphabetically',
+  LENGTH = 'length',
```

Comment:
[checklist] - Enum constant names usually start with capital letter. Values can have any case that's necessary.